Adding some getting started to the README.

mirkoRainer opened the pull request:
Added some small steps for others that may want to get started with this bot.

TheCraiggers commented on index.js:
```diff
@@ -1,8 +1,9 @@
 const Discord = require('discord.js');
 const client = new Discord.Client();
 
+process.env.DISCORD_TOKEN = "insertDiscordTokenHere"
```
Hmm, is there a reason we can't use environment variables in Windows? (I presume you're doing this in Windows from the changes to the README.) I can probably help you out with that if it's just lack of knowledge.

That said, I dislike this change to the code because it would be really easy to mistakenly forget your token is in here when pushing a change to git. There are bots out there that constantly roam the internet looking for discord tokens and other types of passwords, and they will do nasty things within minutes of you uploading your code. Safer to keep it completely separate- if not in an environment variable, than in another file with a corresponding entry in a .gitignore file. Also, for what it's worth, adding the token directly in the code makes debugging far harder for me...

I'd love to hear your thoughts on this. Also, thanks for contributing! I'm super excited that this project got its first PR!

mirkoRainer replied:
Another bot I've toyed with used a json config file that was in the .gitignore for the token. Would that suit you better? I appreciate a good ENV variable though. Just not running on MacOS anymore, although my VM hosting ill be Ubuntu.....

And excited to be the first! I want to use this run a Discord server with Play by Post.

mirkoRainer replied:
Also, are you on discord? I thought I found your repo via Discord. I'm at mirkoRainer#3102

mirkoRainer replied:
One more thing.
Would you be cool with a testing framework for this?
Mocha or Chai?

TheCraiggers replied:
OMG yes. There's actually an open issue for figuring out how to add a automated testing framework for this. Due to its nature though, I haven't found a suitable method for doing it. I've already accidentally caused regression bugs in the bot with some of my changes though, so I know there's an immediate need for _something_. I haven't heard of those two testing frameworks though. Do you have experience with either?

A JSON file with a .gitignore would be totally fine, if not welcome. I could see that being easier for new users.

TheCraiggers commented on README.md:
```diff
@@ -58,3 +58,13 @@ I've made a couple of design choices that I hope won't bite me later.
 * I don't want a huge backend database. Everything will either be stateless, store state in the discord guild itself, or derive state from previous messages. This may make bot interactions a tad slower (although hopefully not noticable), but will hopefully dramatically ease my job if I have to scale this later.
 * Bot commands should be human readable. Saving keystrokes in bot commands is only nice if your players can remember the archaic syntax. This also helps mobile Discord players, as mobile keyboards assume you're trying to type English. Shortcuts will be made available for commands that are used repeatedly.
+
+## Get Things Runnings
```
```suggestion
## Get Things Running
```

TheCraiggers commented on README.md:
```diff
@@ -58,3 +58,13 @@ I've made a couple of design choices that I hope won't bite me later.
+* `C:\Users\user\Pathfinder-Discord-Bot> npm install`
+* Update Line 4 `index.js` with your Discord Token
+* `C:\Users\user\Pathfinder-Discord-Bot> node .\index.js`
+* Add bot to server. Lots of tutorials out there but generally will use n invite url like: `https://discordapp.com/api/oauth2/authorize?client_id=<BOTID>2&permissions=8&scope=bot`
```
```suggestion
* Add bot to server. Lots of tutorials out there but generally will use an invite url like: `https://discordapp.com/api/oauth2/authorize?client_id=<BOTID>2&permissions=8&scope=bot`
```

TheCraiggers reviewed (changes requested):
Thanks for being the first person to submit a PR! I do have one question though, see my long comment.

mirkoRainer commented:
I’m not familiar with JS much. But those were the top google results. Lol. I am familiar with testing though so will take a crack at that.
After I get the JSON config going.

-GK

mirkoRainer commented:
okay. I dumped a bunch of changes into this. Feel free to start blasting away with questions/comments/concerns. I may have gone too deep with this PR. Just got a bit excited....

TheCraiggers committed:
Emergency fix for damage command healing instead

mirkoRainer committed:
Merge branch 'master' of https://github.com/TheCraiggers/Pathfinder-Discord-Bot

mirkoRainer committed:
Updated gitignore
source:
https://github.com/microsoft/TypeScript-Node-Starter/blob/master/.gitignore

mirkoRainer committed:
npm update

mirkoRainer committed:
Fix type checks for PartialMessage

mirkoRainer committed:
Help message extracted to TS file.

mirkoRainer commented:
Started on the main omni file. I think breaking pieces off and converting them one by one into TS will make it easier in the long run.
Did confirm the help messages still work.

Also, should I merge in anything new in another branch aside from master?

mirkoRainer committed:
Adding a Discord Mock for test

mirkoRainer commented:
Instead of hacking the omni.js file into pieces, I'm going to focus on testing it first. Then the TS conversion should be a bit smoother and I'll be more comfortable changing things.

mirkoRainer commented:
Also updated the target branch so I don't mess with the main Juju of your repo.

mirkoRainer committed:
First test setup.
Mocking needs to be tested but it compiles and no error is thrown yet...
